dev/Dockerfile

```diff
@@ -23,14 +27,27 @@ COPY . .
 
 RUN ./dev/gen-anvil-state
 
-FROM ghcr.io/foundry-rs/foundry
+FROM ghcr.io/foundry-rs/foundry:v${FOUNDRY_VERSION}
 
 LABEL maintainer="eng@ephemerahq.com"
 LABEL source="https://github.com/xmtp/smart-contracts"
 LABEL description="XMTP Contracts Image"
 
+WORKDIR /blockchain
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y curl
+
 EXPOSE 8545
 
-COPY --from=builder /build/anvil-state.json anvil-state.json
+COPY --from=builder /build/anvil-state-load-data.json ./anvil-state-load-data.json
+COPY --from=builder /build/dev/start-anvil-with-state ./start-anvil-with-state
+# COPY --from=builder /build/dev/start-anvil-and-load-state ./start-anvil-and-load-state
+
+RUN chmod +x ./start-anvil-with-state
+# RUN chmod +x ./start-anvil-and-load-state
 
-ENTRYPOINT [ "anvil", "--host", "0.0.0.0", "--mixed-mining", "--block-time", "1", "--load-state", "anvil-state.json" ]
\ No newline at end of file
+ENTRYPOINT ["./start-anvil-with-state"]
```

Comment:
this is literally just `anvil`, you dont need either of those `start-*` files. you can just call anvil here.

Reply:
True, but I was hoping to preserve the ability to switch to the other state loading script. I'll stash and clean this all up.